### DISCUSSION TITLE
SO 2.FOREEE question - Elastic Agent

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#8916] [Unsupported Versions] [answered]
Hi,

We are so hyped about the upcoming v2.4! 
Are you going to replace Wazuh with Elastic Agent or it will be just complimentary addition to the security stack ?
Will https://github.com/elastic/detection-rules be utilized somehow ? 

Cheers!
PY

**Comment by dougburks** [comment, marked as answer]
We'll provide more information as we get closer to release.
